<?xml version="1.0" encoding="UTF-8" ?>
<launch>

  <arg name="map"         default="vrep" />
  <arg name="algorithm"   default="CR"/>     
  <arg name="num_robots"  default="2" />
  <arg name="interactive" default="false"/>
  
    <!-- global parameters -->
  <param name="/goal_reached_wait"   value="0"/>
  <param name="/communication_delay" value="0"/> 
  
  <arg name="map_graph_filename" value="$(find patrolling3d_sim)/maps/$(arg map)/$(arg map).graph" />

  <node name="monitor" pkg="patrolling3d_sim" type="monitor" args="$(arg map_graph_filename) $(arg algorithm) $(arg num_robots) $(arg interactive)"  output="screen">
      <param name="build_graph_event_topic" value="/build_graph_event" />
      <param name="graph_topic" value="/patrolling/graph" />
  </node> 

</launch>

